css/portal-dashboard/student-answers.less

```diff
@@ -21,7 +21,7 @@
     min-width: fit-content;
     border-style: solid;
     border-width: 0 0 1px 0;
-    border-color: @cc-charcoal-light2;
+    border-color: @cc-teal-light6;
```

Comment:
Hmmm.... I'm not seeing a change in the spec.  The borders around student answer rows should be cc-charcoal-light2.  I don't think we want to make this change.

Reply:
That's because I was confused. In Zeplin, when you highlight a student box, and then hover over the line at the bottom, it comes back teal light6. But if you just click on a line when the student box is not selected, it's charcoal light2

Reply:
I'll have to look more closely at the spec.  I didn't notice the hover change so there might be some extra styling to add eventually.